server/controllers/userController.js

```diff
@@ -1,53 +1,66 @@
 const ApiError = require('../error/ApiError')
 const bcrypt = require('bcrypt')
 const jwt = require('jsonwebtoken')
-const {User, Orders} = require('../models/models')
+const User = require('../models/user')
+const Orders = require('../models/order')
 
-const generateJws = (id, email, role) => jwt.sign(
+const generateJwt = (id, email, role) => jwt.sign(
     {id, email, role},
     process.env.SECRET_KEY,
     {expiresIn: '24h'}
 )
 
 class UserController {
     async registration(req, res, next) {
-        const {email, password, role} = req.body
-        if (!email || !password) {
-            return next(ApiError.badRequest('Некорректный email или password!'))
-        }
+        try {
+            const {email, password, role} = req.body
+            if (!email || !password) {
+                return next(ApiError.badRequest('Некорректный email или password!'))
+            }
 
-        const candidate = await User.findOne({where: {email}})
-        if (candidate) {
-            return next(ApiError.badRequest('Пользователь с таким email уже существует!'))
-        }
+            const candidate = await User.findOne({where: {email}})
+            if (candidate) {
```

Comment:
candidate is not the best name, maybe doesUserExist or something like this, or even just user